README.md

```diff
@@ -49,7 +50,43 @@ Similarly, [calorimetric.json](https://github.com/building-envelope-data/api/blo
 
 With you web browser, you can search our [wiki](https://github.com/building-envelope-data/api/wiki), the [issues](https://github.com/building-envelope-data/api/issues) and [pull requests](https://github.com/building-envelope-data/api/pulls) and contribute to them.
 
-In order to browse the code conveniently, you should first [clone the repository](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/cloning-a-repository) and then use a text editor, for example [Visual Studio Code](https://code.visualstudio.com/).
+### In your browser
```

Comment:
I'd call it "web browser" (there are also file browsers and other kinds of browsers).

Reply:
Resolved by [ed9d673](https://github.com/building-envelope-data/api/pull/336/commits/ed9d673a72d69c2ae3d6477170901424afa06295)